src/onboarding/registration/MockedStatSigFeatureFlag.ts

```diff
@@ -0,0 +1,6 @@
+export const shouldShowAlternatePlaceholder = () => {
```

Comment:
Would this file be removed when this starts using statsig? If so, could we just inline this directly in NameAndPicture.tsx so it's easier to remove or at least add a TODO here?

Reply:
I'll add a TODO here.